adal/build.gradle

```diff
@@ -22,6 +22,7 @@ android {
 
     defaultConfig {
         minSdkVersion 14
+        //noinspection OldTargetApi
         targetSdkVersion 23
```

Comment:
ah nevermind. my bad!

---
In reply to: [73989248](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/694#discussion_r73989248) [](ancestors = 73989248,73977550)